### ISSUE TITLE
Font / Icon query (question, not a bug!)

### ISSUE THREAD

**Issue by paul-jewell** [alphapapa/hammy.el#20]
First of all, many thanks for making this available for all!
Quick question: I have installed hammy in emacs 29.1 on Gentoo, and I am not getting the font icons showing correctly - I am simply getting the Unicode boxes. For example, instead of : "Hammy (🍅): Break time!", "I get Hammy ([01F345]): Break time!", with the unicode box showing in the standard emacs format (box with 01F over the 345).

Of course, this is **not** an issue with your code, but I am sure the problem probably lies in my setup. I have tried the following to resolve the problem:
- Installed nerd-icons
- Tried the nerdfont version of firacode (I normally use iosevka)
- Tried many other fonts to see if anyof them have the correct code points defined.

Note: all-the-icons are already installed for dired, and work as expected. 

I would love to know what fonts / icons you are using to get the icons correctly displayed in emacs. If you let me know, I would be happy to issue a pull request against the README to explain what to do if you come across the problem I have.

Many thanks!
Paul

**Comment by alphapapa** [comment]
Hi Paul,

Thanks for the kind words.  I hope it's useful to you.

About the emojis, the best knowledge I have about a solution is in the Ement.el readme here: https://github.com/alphapapa/ement.el#displaying-symbols-and-emojis  However, AFAIK Emacs 29 isn't supposed to need those workarounds anymore.  So it may have something to do with other parts of your system config.  FWIW, I use Debian, and it works for me.

**Comment by paul-jewell** [comment]
Hi Adam, 
Happy New Year!
I agree with your comments about my configuration. I will start with an empty one, and add stuff in to see if I can identify the problem, then post the solution here.

**Comment by paul-jewell** [comment]
Ok, so the issue related to configuration of font handling for my user account on this machine. Some time (long ago) I added a local fontconfig file which interfered with the font provision to emacs. Once this file was removed, the problem disappeared, and the correct icons appear, without any configuration in emacs 29.1. 
So if you are experiencing a problem with emoji presentation on your system, then I recommend you check there isn't a file in ~/.config/fontconfig which is impacting your font selection.
Thanks again to Adam for this software, and also the other packages he generously makes available to the community.
